chromadb/segment/impl/distributed/segment_directory.py

```diff
@@ -242,7 +242,9 @@ def stop(self) -> None:
     def get_segment_endpoint(self, segment: Segment) -> str:
         if self._curr_memberlist is None or len(self._curr_memberlist) == 0:
             raise ValueError("Memberlist is not initialized")
-        assignment = assign(segment["id"].hex, self._curr_memberlist, murmur3hasher)
+        assignment = assign(segment["id"].hex, self._curr_memberlist, murmur3hasher, 1)[
+            0
+        ]
         service_name = self.extract_service_name(assignment)
         assignment = f"{assignment}.{service_name}.{KUBERNETES_NAMESPACE}.{HEADLESS_SERVICE}:50051"  # TODO: make port configurable
         return assignment
```

Comment:
Since we are planning to assign by collection id instead of segment id, I'm wondering if we should put this somewhere else?

Reply:
I don't understand what that has to do with this PR.

Reply:
Yes this has nothing to do with PR. Marked as resolved.